Makefile

```diff
@@ -1,60 +1,21 @@
-BINARY := docker-machine-driver-oxide
+GO = go
 
-BUILD=`date +%FT%T%z`
+BINDIR := bin
+BINARY := oxide-rancher-machine-driver
 
-LDFLAGS=-ldflags "-w -s -extldflags '-static'"
+LDFLAGS := -ldflags "-w -s -extldflags '-static -Wl,--fatal-warnings'"
+TAGS := "netgo osusergo no_stage static_build"
```

Comment:
Do the Rancher packages required these tags? Just curious what they are needed for.

Reply:
No. We have a standard on the distros team on how we build (static) Go binaries and I just copy pasted over to here. Go will still link against the OS if not specifiied.